python/backtracking/subset_sum.py

```diff
@@ -0,0 +1,25 @@
+def SubsetSum(set, n, sum) : 
+    if (sum == 0) : 
+        return True
+    if (n == 0 and sum != 0) : 
+        return False
+    if (set[n - 1] > sum) : 
+        return SubsetSum(set, n - 1, sum); 
+        
+    return SubsetSum(set, n-1, sum) or SubsetSum(set, n-1, sum-set[n-1]) 
+      
+      
+
+n=int(input("number of elements :- "))
+sum=int(input("Enter the sum:- ")) 
+print("elements in the set:-")
```

Comment:
Please comment out a sample input given so that it makes this easy for others to try and understand as well. Thankyou.